### PR TITLE
Add browser compat for <a> element

### PR DESCRIPTION
**Pull request by teoli2003**
This is an important element, it deserves its own PR :-)

### CODE REVIEW

File: html/elements/a.json [hunk 1]
```diff
@@ -0,0 +1,771 @@
...
+            },
+            "status": {
+              "experimental": false,
+              "standard_track": true,
+              "deprecated": false
+            }
+          },
+          "href_top": {
+            "__compat": {
+              "description": "<code>href = '#top'</code> behavior.",
```
Comment (Elchi3) [inline html/elements/a.json]: Descriptions shouldn't be sentences (no full stop), they should rather be labels. I would recommend `"<code>href='#top'</code>"`.

**Comment by teoli2003** [comment]
Fixed.

File: html/elements/a.json (hunk 1 above)
Comment (Osama2035) [reply html/elements/a.json]: Osama

**Review by Osama2035 — COMMENTED** [review]
[](url)

**Pull request merged**